src/convertToObject.js

```diff
@@ -1,18 +1,27 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const string = sourceString.split('\n')
```

Comment:
Try to provide more meaningful names instead of `result` and `string`.
Actually, it's not even string but array